bazel/BUILD

```diff
@@ -149,6 +149,11 @@ config_setting(
     values = {"define": "boringssl=fips"},
 )
 
+config_setting(
+    name = "enable_quiche",
+    values = {"define": "enable_quiche=enabled"},
```

Comment:
"enable_quiche=enabled" sounds a little redundant. How about just "quiche=enabled"?

Reply:
Done in ##5838